test/cypress/tests/ui/DataEmpty.cy.ts

```diff
@@ -0,0 +1,71 @@
+import { createEditorWithTextBlocks } from '../../support/utils/createEditorWithTextBlocks';
+
+describe('inputs [data-empty] mark', function () {
+  it('should be added to the editor on initialization', function () {
+    createEditorWithTextBlocks([
+      'First', // not empty block
+      '', // empty block
+    ]);
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .first()
+      .should('have.attr', 'data-empty', 'false');
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .last()
+      .should('have.attr', 'data-empty', 'true');
+  });
+
+  it('should be added to the block on input', function () {
+    createEditorWithTextBlocks([
+      'First', // not empty block
+      '', // empty block
+    ]);
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .last()
+      .type('Some text');
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .last()
+      .should('have.attr', 'data-empty', 'false');
+  });
+
+  it('should be added to the block on focus', function () {
```

Comment:
It's good to cover this case, but if this behaviour is broken, this case will pass because of adding this attr by default on initialisation. I don't know how to test it in the right way.

Reply:
removed this test